echo.go

```diff
@@ -301,6 +304,13 @@ var (
 	}
 
 	MethodNotAllowedHandler = func(c Context) error {
+		// 'Allow' header RFC: https://datatracker.ietf.org/doc/html/rfc7231#section-7.4.1
+		// >> An origin server MUST generate an Allow field in a 405 (Method Not Allowed) response
+		//    and MAY do so in any other response.
+		routerAllowMethods, ok := c.Get(ContextKeyHeaderAllow).(string)
+		if ok && routerAllowMethods != "" {
```

Comment:
This is here to avoid adding empty header. As this handler is at the end of middleware chain there could be "actors" set it to empty (Echo does not have way to delete/unset context key)

Reply:
Ist there a way we could disable a resource, meaning route is defined, but no options are allowed as described as a special usecase in the RFC?